test/screenshot/spec/mdc-top-app-bar/classes/baseline-always-closed-with-action-items.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
Name this file name `baseline-short-collapsed.html` 

In general, these are named after class names / modifiers.

Reply:
This is the name of it in the README.md - see `Short - Always Closed`. Do you want to also update it in the README?

Reply:
We don't have to change the README. 

I think, having test file names matching the class names finds it easy to discover what this is covering.